api/LinkStyle.json

```diff
@@ -52,10 +52,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/LinkStyle/sheet",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
`LinkStyle` is an interface mixin. One of the interfaces that include it is `HTMLLinkElement`, and this is supported in Chromium:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/html_link_element.idl?l=48&rcl=49cdaf07257db8ae0bd10653a44d3221af479145

https://github.com/mdn/browser-compat-data/issues/472 discusses the difficulties with mixins, and what to do if the support isn't same same for all real interfaces. I would suggest either reverting all changes to mixins, or setting values to true/false only if all of the real interfaces have the same status.